features/step_definitions/supplier_steps.rb

```diff
@@ -68,6 +68,10 @@
   puts "service id: #{@service['id']}"
 end
 
+Given 'I ensure that all update audit events are acknowledged' do
+  acknowledge_all_service_updates
+end
+
 Given 'I ensure that all update audit events for that service are acknowledged' do
```

Comment:
This step is only used in [one other place](https://github.com/alphagov/digitalmarketplace-functional-tests/blob/76ce67b17f711f2fb904c35040b55c212161485b/features/supplier/view_and_edit_g_cloud_services.feature#L12) - is it worth changing that step to your new one too, and getting rid of this one altogether?

Reply:
I can't think of any reason not to... do you have any thoughts @risicle?

Reply:
That would abandon any attempt at making these tests isolatable, which I did quite intentionally. It's a pity that they will eventually fall off the bottom of the 100 edits. What about making the other tests which make service edits call `I ensure that all update audit events for that service are acknowledged` when they're done?

Reply:
@risicle does that comment extend to the whole PR?

Reply:
More or less, yes.